docs/customization.md

```diff
@@ -210,6 +210,33 @@ i18n:
     - de
     - it
   defaultLocale: en # Optional. Used as fallback when browser language preferences don't match supported locales, or defaults to 'en' if not specified.
+  overrides: # List of json translation files to override existing plugin translations or add translations for a locale
+    - <path-to>/<plugin-x>.json
+    - <path-to>/<plugin-y>.json
```

Comment:
nit:

The example plugin-x, plugin-y might give the impression that overrides are limited to individual plugins. Since overrides can also be more general, maybe we could adjust the example to avoid confusion.
 
```suggestion
  overrides: # List of JSON translation files applied in order (last file wins).  Each file may override/add translations for one or more plugins/locales
    - <path-to>/<overrides-1>.json
    - <path-to>/<overrides-2>.json
```
This way, the documentation makes it clear that overrides are not restricted to plugin-level only.